pkg/migration/controllers/migration.go

```diff
@@ -1123,12 +1123,12 @@ func (m *MigrationController) preparePVCResource(
 	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(object.UnstructuredContent(), &pvc); err != nil {
 		return err
 	}
-
+	sc := ""
```

Comment:
Was there any other reason why we were originally setting the StorageClass name to empty? We did that change in 2.6.4 to keep the PVC sizes in sync by comparing the hashes.